fortress/ros2_gazebo_integration_tutorial.md

```diff
@@ -0,0 +1,111 @@
+# ROS 2 and Gazebo Integration Tutorial
```

Comment:
I only read the Fortress version of this file, but the Harmonic version should be updated for these comments too

Reply:
Updated Garden and Harmonic based on the comments